kyo-prelude/shared/src/main/scala/kyo/Poll.scala

```diff
@@ -249,17 +213,17 @@ object Poll:
                         // This creates the demand-driven cycle between emit and poll
                         ArrowEffect.handleFirst(pollTag, poll)(
                             handle = [C2] =>
-                                (ack, pollCont) =>
+                                (unit, pollCont) =>
```

Comment:
I think using `_` here and an explicit `()` bellow would be more clear. I wasn't sure if this was the regular unit value or something else initially